client/executeClientAction.go

```diff
@@ -65,6 +65,7 @@ func runScrapeWithInfo(ctx context.Context, jsAction string, url string) string
 		// Navigate to user's page
 		chromedp.Navigate(url),
 		// Execute url grabber script
+		chromedp.WaitReady("video"),
```

Comment:
Please remove this.
This will force waiting for the video even if the video does not exist.
I've added some error handling since your fork that covers this case.

Reply:
done